openpype/hosts/unreal/lib.py

```diff
@@ -369,11 +369,11 @@ def get_compatible_integration(
 
 def get_path_to_cmdlet_project(ue_version: str) -> Path:
     cmd_project = Path(
-        os.path.abspath(os.getenv("OPENPYPE_ROOT")))
+        os.path.abspath(os.getenv("AYON_UNREAL_ROOT")))
```

Comment:
Wouldn't this be easier?
```suggestion
        os.path.dirname(os.path.abspath(__file__)))
```